receiver/snmpreceiver/testdata/integration/v3_config_expected_metrics.json

```diff
@@ -0,0 +1,117 @@
+{
+  "resourceMetrics": [
+    {
+      "scopeMetrics": [
+        {
+          "scope": {
+            "name": "otelcol/snmpreceiver",
+            "version": "latest"
+          },
+          "metrics": [
+            {
+              "description": "System uptime",
+              "name": "snmp.test.metric.sysuptime",
+              "gauge": {
+                "dataPoints": [
+                  {
+                    "aggregationTemporality": "AGGREGATION_TEMPORALITY_CUMULATIVE",
```

Comment:
This seems wrong. Gauge metrics should not have a value for aggregation temporality. I don't see anything wrong with the test.

@StefanKurek, any idea what's going on here? Might be a bug in the scraper.

Reply:
@djaglowski it sounds like @tamir-michaeli manually created the response, so i believe this is just a problem with the response.